package-res/resources/web/prompting/api/UiAPI.js

```diff
@@ -15,8 +15,25 @@
  *
  */
 
+/**
+ * The Prompting UI API Class
+ * Provides general functions for working with UI of prompting.
+ *
+ * @name UiAPI
+ * @class
+ */
 define([], function() {
   return function(api) {
-
+    /**
+     * Makes visible the progress indicator by calling function Dashboard#showProgressIndicator.
```

Comment:
I would change this line to "Makes the progress indicator visible by calling Dashboard#showProgressIndicator"